lib/eclipse/Parser/Parser.cpp

```diff
@@ -229,6 +229,9 @@ class ParserState {
 
     public:
         std::shared_ptr< RawKeyword > rawKeyword;
+        ParserKeywordSizeEnum lastSizeType;
```

Comment:
The `lastSizeType` variable should be initialized to `SLASH_TERMINATED` - to ensure well-defined behaviour for parsing errors in the start of the input stream.

Reply:
The initialization should be here - i.e.:
```C++
ParserKeywordSizeEnum lastSizeType = SLASH_TERMINATED;
```